.travis.yml

```diff
@@ -2,46 +2,54 @@ sudo: false
 language: python
 matrix:
   include:
-    - python: "2.7"
```

Comment:
`$ travis encrypt` CLI tool reformatted these lines. The indentation for yaml differs.